xbmc/pvr/addons/PVRClient.cpp

```diff
@@ -482,7 +493,13 @@ const std::string& CPVRClient::GetConnectionString() const
 
 const std::string& CPVRClient::GetFriendlyName() const
 {
-  return m_strFriendlyName;
+  /* display name = backend name:connection string */
+  const std::string strFriendlyName =
+      GetConnectionString().empty() // connection string is optional
+          ? GetBackendName() != g_localizeStrings.Get(13205) ? GetBackendName() : Name()
```

Comment:
This does not work. String 13025 is "Unknown" (capital U) and `DEFAULT_INFO_STRING_VALUE` is "unknown" (small u"). And in other languages the two strings may differ even more. For de_de 13205 is "Unbekannt", for example. Even though both strings would be accidentally equal, this code would break as soon as somebody either changes the value of `DEFAULT_INFO_STRING_VALUE` or string 13205. 

What you probably want to do is `GetBackendName() != DEFAULT_INFO_STRING_VALUE`?

Reply:
Yes is what I wanted to do, though about it while hiking in the snow and couldn't change it.